src/components/Card/Card.tsx

```diff
@@ -9,19 +9,19 @@ import classes from './Card.module.scss';
  }
 
 const Card: FC<CardProps> = ({title, data}) => {
-      return (
```

Comment:
this file has nothing to do with your PR, revert changes and leave this one how it's done on master branch

Reply:
file restored accordin master